test/hotspot/jtreg/runtime/reflect/ReflectOutOfMemoryError.java

```diff
@@ -25,7 +25,7 @@
  * @test
  * @bug 8297977
  * @summary Test that throwing OOM from reflected method gets InvocationTargetException
- * @run main/othervm ReflectOutOfMemoryError
+ * @run main/othervm/timeout=150 ReflectOutOfMemoryError
```

Comment:
Doesn't it make a sense to exclude this test from tier1, if it might took more than 2 minutes to complete?

Reply:
It's not that simple. Here's my comment from the bug:

https://bugs.openjdk.org/browse/JDK-8307783?focusedCommentId=14582995&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14582995

In jdk-21+24-2010 (which has finished Tier[1-7]), there are 79 passing runs of 
this test and the times range from 31s -> 7m 37s. The slowest is on linux-arch64, 
but the fastest linux-aarch64 is 44s so it's not simply that linux-aarch64 is slow.

Reply:
Do we know why this test do not set Xmx to some reasonable value?

Reply:
filed https://bugs.openjdk.org/browse/JDK-8316060